app/models/sushi.rb

```diff
@@ -163,20 +163,28 @@ module MetricTypeCoercion
       # "Unique_Title_Requests"
     ].freeze
 
-    def coerce_metric_types(params = {})
+    ALLOWED_METRIC_TYPES_PLATFORM_USAGE = [
+      # the platform usage report only contains requests. see https://countermetrics.stoplight.io/docs/counter-sushi-api/mgu8ibcbgrwe0-pr-p1-performance-other for details
+      "Unique_Item_Requests",
+      "Total_Item_Requests"
+    ].freeze
+
+    def coerce_metric_types(params = {}, platform_usage_report = false)
```

Comment:
The boolean for `platform_usage_report` is asking for trouble.  Namely we need to know what it means.

What if we were to instead adopt a different approach, namely adding a class method that returns the allowed metric types.  Then for the platform usage report, we overwrite that method.

I can pair on this to highlight the approach.